README.md

```diff
@@ -150,27 +150,28 @@ See also:
-| CustomSeparatedWithNamesAndTypes           | ❌            | ✔️             | ❌            | ✔️            |
+| Format                                     | Input (array) | Input (stream) | Input (object) | Output (JSON) | Output (text) |
+| ------------------------------------------ | ------------- | -------------- | -------------- | ------------- | ------------- |
+| JSON                                       | ❌            | ❌             | ✔️             | ✔️            | ✔️            |
+| JSONObjectEachRow                          | ❌            | ❌             | ✔️             | ✔️            | ✔️            |
+| JSONEachRow                                | ✔️            | ✔️             | ❌️            | ✔️            | ✔️            |
+| JSONStringsEachRow                         | ✔️            | ✔️             | ❌️            | ✔️            | ✔️            |
+| JSONCompactEachRow                         | ✔️            | ✔️             | ❌️            | ✔️            | ✔️            |
+| JSONCompactStringsEachRow                  | ✔️            | ✔️             | ❌️            | ✔️            | ✔️            |
+| JSONCompactEachRowWithNames                | ✔️            | ✔️             | ❌️            | ✔️            | ✔️            |
+| JSONCompactEachRowWithNamesAndTypes        | ✔️            | ✔️             | ❌️            | ✔️            | ✔️            |
+| JSONCompactStringsEachRowWithNames         | ✔️            | ✔️             | ❌️            | ✔️            | ✔️            |
+| JSONCompactStringsEachRowWithNamesAndTypes | ✔️            | ✔️             | ❌️            | ✔️            | ✔️            |
+| CSV                                        | ❌            | ✔️             | ❌             | ❌            | ✔️            |
+| CSVWithNames                               | ❌            | ✔️             | ❌             | ❌            | ✔️            |
+| CSVWithNamesAndTypes                       | ❌            | ✔️             | ❌             | ❌            | ✔️            |
+| TabSeparated                               | ❌            | ✔️             | ❌             | ❌            | ✔️            |
+| TabSeparatedRaw                            | ❌            | ✔️             | ❌             | ❌            | ✔️            |
+| TabSeparatedWithNames                      | ❌            | ✔️             | ❌             | ❌            | ✔️            |
+| TabSeparatedWithNamesAndTypes              | ❌            | ✔️             | ❌             | ❌            | ✔️            |
+| CustomSeparated                            | ❌            | ✔️             | ❌             | ❌            | ✔️            |
+| CustomSeparatedWithNames                   | ❌            | ✔️             | ❌             | ❌            | ✔️            |
+| CustomSeparatedWithNamesAndTypes           | ❌            | ✔️             | ❌             | ❌            | ✔️            |
```

Comment:
Let's move this table to the docs section on the website https://github.com/ClickHouse/clickhouse-docs